tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Debugger.cs

```diff
@@ -4,40 +4,26 @@
 // </copyright>
 
 using Datadog.Trace.Debugger;
-using Datadog.Trace.Logging.DirectSubmission;
 
 namespace Datadog.Trace.Configuration
 {
     internal static partial class ConfigurationKeys
     {
         internal static class Debugger
         {
-            /// <summary>
-            /// Configuration key for debugger poll interval (in seconds).
-            /// </summary>
-            /// <seealso cref="DebuggerSettings.ProbeConfigurationsPollIntervalSeconds"/>
-            public const string PollInterval = "DD_DEBUGGER_POLL_INTERVAL";
-
             /// <summary>
             /// Configuration key for the URL used to query our backend directly for the list of active probes.
             /// This can only be used if DD_API_KEY is also available.
```

Comment:
```suggestion
            /// Used to send Live Debugger snapshots directly to our backend.
            /// This can only be used if DD_API_KEY is also available.
```